airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.2.2'
     features = ['db-sources']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
I'll undo all these `useLocalCdk = true` after publishing the CDK, after approval.